src/renderer/src/components/browser-pane/remote-browser-legacy-input.test.ts

```
@@ -0,0 +1,115 @@
+    scroller.style.overflowY = 'auto'
+    scroller.append(child)
+    document.body.append(scroller)
+    Object.defineProperties(scroller, {
+      scrollHeight: { value: 500 },
+      clientHeight: { value: 100 }
+    })
+    const scrollBy = vi.fn()
+    scroller.scrollBy = scrollBy
+    document.elementFromPoint = vi.fn(() => child)
+
+    window.eval(buildLegacyRemoteBrowserWheelExpression(10, 20, 0, 120))
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)